vault/resource_gcp_auth_backend_test.go

```diff
@@ -46,6 +48,7 @@ func TestGCPAuthBackend_basic(t *testing.T) {
 			{
 				Config: testGCPAuthBackendConfig_update(path, gcpJSONCredentials),
 				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckAuthMountExists("vault_gcp_auth_backend.test", &resAuthFirst),
```

Comment:
I wonder if moving this to `testutil.go` would make sense instead of keeping it in the github auth resource test file?

Reply:
Hey Austin,

I like your suggestion, but would you be open to capturing that work in a separate Github issue or addressing it as part of https://github.com/hashicorp/terraform-provider-vault/issues/882 (which I'd be more than happy to pick up and was planning on doing so!). The reason I suggest that is, should we pursue this, I believe the other helper functions linked [here](https://github.com/hashicorp/terraform-provider-vault/blob/main/vault/resource_auth_backend_test.go#L274-L333) that also assert against the status of an auth mount) should also be relocated to `./testutil/testutil.go`.

Are you ok if I refactor the auth mount test logic in `./testutil/testutil.go` as part of https://github.com/hashicorp/terraform-provider-vault/issues/882 and we go ahead and merge this as is? Let me know, and thank you so much for the quick review!

Reply:
Thanks for the detailed reply, @jdelnano! That sounds reasonable to me 👍